src/documentation/tooling/index.md

```diff
@@ -0,0 +1,208 @@
+# Documentation tooling
+
+This section provides details on *how* documentation for the research project,
+and the **Hexatomic** software as well as the infrastructure used to develop and
+provide it, is facilitated.
+
+## Requirements
+
+The requirements for documentation tooling are specific to our research project
+in some respects (e.g., pertaining to Javadoc), but all others can be applied
+to generic (research) software project setups. And those that are 
+project-specific can probably be transferred to other use cases relatively 
+easily.
+
+> **Requirements summary**
+> 
+> The requirements for documentation tools for sustainable software development are
+> 
+> - (1) Sustainability
+> - (2) Single tool toolchain
+> - (3) Usability  
+>     - (3a) Human-readable source format  
+>     - (3b) Javadoc integration  
+>     - (3c) Continuous integration capabilities  
+>     - (3d) Maintainability
+>     - (3e) Maintainability (dependencies)
+>     - (3f) Usability of representations
+> - (4) Different representation forms  
+
+### Sustainability
+
+The [documentation sustainability](../sustainability/) section establishes that
+
+> [documentation] sustainability must be ensured by choosing sustainable tooling 
+[...].
+
+Software sustainability covers many aspects, including community parameters such
+as size, number and frequency of contributions; whether a code is open source;
+the programming language it is implemented with; ease of installation; its
+dependency tree, etc.
+
+Accordingly, there is currently no canonic definition of software 
+sustainability, as discussed in [1]. One of the definitions given in [1] seems
```

Comment:
Footnote formatting should be using the markdown syntax for footnotes.

Reply:
It's not a footnote but an IEEE formatted reference. I found it better to include any references at the bottom of the respective paragraph.